daemon/cmd/daemon_main.go

```diff
@@ -1359,6 +1359,12 @@ func initEnv(cmd *cobra.Command) {
 		}
 	}
 
+	if option.Config.IPAM == ipamOption.IPAMNone {
+		if option.Config.LocalRouterIP == "" {
+			log.Fatalf("%s must be specified when %s=%s", option.LocalRouterIP, option.IPAM, ipamOption.IPAMNone)
```

Comment:
```suggestion
			log.Fatalf("%s must be specified when %s=%s.", option.LocalRouterIP, option.IPAM, ipamOption.IPAMNone)
```

to conform with the rest of the log msgs

Reply:
FWIW, it seem far more common overall to skip the `.`:

```
$ git grep "Fatal.*[^.]\"," | wc -l
494
$ git grep "Fatal.*\.\"," | wc -l
14
```

(I don't have any particular preference for either approach but it seems like this is neither here nor there as a nit)